core/manager.py

```diff
@@ -1154,6 +1157,17 @@ def startModule(self, base, key):
             If the module is already loaded, just activate it.
             If the module is an active GUI module, show its window.
         """
+
+        if QtCore.QThread.currentThread() is not self.thread():
+            QtCore.QMetaObject.invokeMethod(
```

Comment:
Might this cause the trouble I mentioned?

Reply:
Thanks for the tests, I think I will need to double check on Windows. I will then come back to you.

Reply:
I deleted this part of the pull request because it is not easily solvable in the old core. New core handles it automatically.